src/mds/mdstypes.h

```diff
@@ -1087,11 +1087,7 @@ class feature_bitset_t {
     return *this;
   }
   bool empty() const {
-    for (auto& v : _vec) {
-      if (v)
-	return false;
-    }
-    return true;
+    return _vec.empty();
```

Comment:
This isn't quite the same: `block_type` is a `uint64_t`. If the vector is only composed of `0`s, then it's still "empty".

Suggest you add a comment so the next person reading the code is not also confused.

Reply:
I see, that's the definition of empty here. So it's like an `std::all_of`.